### ISSUE TITLE
🔜 GitHub Action

### ISSUE THREAD

**Issue by gr2m** [skypackjs/package-check#4]
I'm working on creating an official GitHub Action to integrate Skypack's package-score into your CI. I'll keep y'all posted here 👇🏼

**Comment by stramel** [comment]
I just added a quick run to my existing checks for now. https://github.com/react-hook-form/react-hook-form/pull/3236/files#diff-5ca9504d40891cc08c8fff3c5d9be551bb2a39bd09803540bff22c36e7e7b8ea

**Comment by gr2m** [comment]
Yes you could to the same workflow as above, just exchange `uses: skypackjs/package-check@v1` with `npx @skypack/package-check`. But with an action we have more possibilities than a simple success / fail status

**Comment by gr2m** [comment]
Here is a first version: could someone give it a try? 

https://github.com/gr2m/skypack-quality-score-action

I gotta run now, but will check back later tonight

**Comment by gr2m** [comment]
Works as expected: 

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/39992/97214582-8afcad00-1780-11eb-9607-10e176bedd79.png">

@FredKSchott I would suggest to create a separate repository for the action: https://github.com/skypackjs/package-check-action. Once we rename the repository, I'd publish it to the marketplace so folks can find it. Once that's done, we can discuss adding more features to it, like a better report by creating a check run instead of relying on the binary status of the GitHub Action run

**Comment by FredKSchott** [comment]
Awesome! Repo created, just gave you access: https://github.com/skypackjs/package-check-action
